src/components/HathorModal.js

```diff
@@ -38,9 +38,9 @@ const styles = StyleSheet.create({
     borderRadius: 8,
     alignItems: 'center',
     paddingHorizontal: 16,
-    paddingBottom: 56,
```

Comment:
Why did you change this? You changed to less than half the original value, what screens are impacted with this?